atom/browser/renderer_host/DEPS

```diff
@@ -0,0 +1,6 @@
+include_rules = [
+  # renderer_host is intended to support the content layer's renderers. No
+  # reference to the tab contents is allowed.
+  "-chrome/browser/tab_contents",
+]
```

Comment:
Are we checking this? This doesn't seem relevant for Electron.

Reply:
No, I removed it,